test.html

```diff
@@ -1,13 +1,15 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf8">
+    <meta charset="utf-8">
     <title>Processing.js</title>
     <script src="processing.js"></script>
   </head>
   <body>
     <h1>Processing.js</h1>
     <script type="text/processing">
+      document.getElementById("canvas");
```

Comment:
I'd like to know, too, this call does nothing - it gets an element by `id`, and then throws it away again.